### PR TITLE
COE-842: allow disabling the ssl verification for ImportData.py

### PR DESCRIPTION
**Pull request by mvoigt-sag**

### CODE REVIEW

File: simulators/extras/ArgumentsAndCredentialsHandler.py [hunk 1]
```diff
@@ -176,10 +179,11 @@ def RemoveTrailingSlashFromBaseUrl(baseUrl):
     return baseUrl
 
 
-def CheckTenantConnection(baseUrl, C8Y_HEADERS):
+def CheckTenantConnection(baseUrl, C8Y_HEADERS, session):
```
Comment (pejo-sag) [inline simulators/extras/ArgumentsAndCredentialsHandler.py:182]: Why is this session added here? This function is also used by the export script. Please adapt

Comment (pejo-sag) [reply simulators/extras/ArgumentsAndCredentialsHandler.py:182]: Or set a default

**Review by pejo-sag — APPROVED** [review]
Just one small mistake. Everything else looks good

File: simulators/extras/ArgumentsAndCredentialsHandler.py (hunk 1 above)
Comment (mvoigt-sag) [reply simulators/extras/ArgumentsAndCredentialsHandler.py:182]: the session is also used to verify the connection; I added a session now also to the ExportProfileData.py 
- unfortunately I didnt find a way to use the session or the VERIFY_SSL_CERTIFICATE variable in the c8y_api to also allow exporting without verifying the SSL certificate

**Review by pejo-sag — APPROVED** [review]

**Pull request merged**